pyproject.toml

```
@@ -35,16 +35,16 @@ classifiers = [
 requires-python = ">= 3.11, < 4"
 dependencies = [
   "python-dateutil >= 2.8.2, < 3.0",
-  "typing-extensions == 4.11.0",
+  "typing-extensions => 4.11.0, < 5.0.0",
   # Make sure to update the version for cross-referencing also in the
   # mkdocs.yml file when changing the version here (look for the config key
   # plugins.mkdocstrings.handlers.python.import)
-  "frequenz-sdk == v1.0.0-rc6",
-  "frequenz-channels == 1.0.0",
+  "frequenz-sdk == 1.0.0-rc601",
```

Woops, we also need to remove the `-`, it is very weird how `pip` accepts a slightly broken version specifier but not a completely broken one! :laughing: 
```suggestion
  "frequenz-sdk == 1.0.0rc601",
```